drawBot/drawBotDrawingTools.py

```diff
@@ -1560,6 +1560,14 @@ def listFontVariations(self, fontName=None):
         """
         return self._dummyContext._state.text.listFontVariations(fontName)
 
+    def listFontVariationNamedInstances(self, fontName=None):
+        """
+        List all named instances for the current font.align
```

Comment:
Can the doc string somehow be shared with baseContext.py?

Reply:
I added now a couple of them

this needs to happen for other methods docs strings.